Tests/SentryCrashTests/SentryFrameRemoverTests.swift

```diff
@@ -60,16 +60,31 @@ class SentryFrameRemoverTests: XCTestCase {
              fixture.nonSentryFrame]
         
         let actual = SentryFrameRemover.removeNonSdkFrames(frames)
-                XCTAssertEqual(frames, actual)
+        XCTAssert(equivalent(expected: frames, actual: actual))
     }
     
     func testNoSdkFrames_NoFramesRemoved() {
         let actual = SentryFrameRemover.removeNonSdkFrames(fixture.nonSentryFrames)
-        XCTAssertEqual(fixture.nonSentryFrames, actual)
+        XCTAssert(equivalent(expected: fixture.nonSentryFrames, actual: actual))
     }
     
     func testOnlySdkFrames_AllFramesRemoved() {
         let actual = SentryFrameRemover.removeNonSdkFrames(fixture.sentryFrames)
-        XCTAssertEqual(fixture.sentryFrames, actual)
+        XCTAssert(equivalent(expected: fixture.sentryFrames, actual: actual))
+    }
+}
+
+private extension SentryFrameRemoverTests {
+    /// Compare elements pairwise to see if the two input arrays are equivalent or not.
+    func equivalent(expected: [Frame], actual: [Frame]) -> Bool {
+        guard expected.count == actual.count else { return false }
+        for i in 0..<expected.count {
+            let nextExpected = expected[i]
+            let nextActual = actual[i]
+            if nextExpected.package != nextActual.package {
```

Comment:
`m`: I guess we shouldn't only just compare the package only. You could use: https://github.com/getsentry/sentry-cocoa/blob/74192858dff8cf729ce015b06fe9d3ff64fe7b74/Tests/SentryTests/Protocol/SentryThreadEquality.swift#L35-L36